code/game/objects/machinery/computer/camera_advanced.dm

```diff
@@ -154,13 +155,18 @@
 
 /obj/machinery/computer/camera_advanced/proc/give_eye_control(mob/user)
 	give_actions(user)
+	RegisterSignal(user, COMSIG_MOB_CLICK_SHIFT, .proc/send_orders)
 	current_user = user
 	eyeobj.eye_user = user
 	eyeobj.name = "Camera Eye ([user.name])"
 	user.remote_control = eyeobj
 	user.reset_perspective(eyeobj)
 	eyeobj.setLoc(eyeobj.loc)
 
+/obj/machinery/computer/camera_advanced/proc/send_orders(atom/object) ///Send an order to marines on the ground
```

Comment:
```suggestion
/obj/machinery/computer/camera_advanced/proc/send_orders(datum/source, atom/object) ///Send an order to marines on the ground
```
the first argument a send_signal() always sends is whatever is sending the signal, meaning the atom object would be the player clicking and not whatever the player clicked on.